neurodiffeq/solvers_utils.py

```diff
@@ -1,12 +1,12 @@
 import dill
 import numpy as np
 from abc import ABC, abstractmethod
+import pathlib
 
 class PretrainedSolver(ABC):

```

Comment:
I don't think we need to mark this class as `ABC`. This is more like a _Mixin_ class, which provides functionality of saving and loading.

Reply:
The save-load-solver branch is currently just our trial and error for the different options of saving to understand the benefits and shortcoming of each method. We can review this branch when we are read to merge save-load-solver branch with the main branch later

Reply:
Oh, sorry. In that case, I'll merge the PR so you can try it out.